src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetAccountEmployerAgreementsRemove/WhenIGetAccountEmployerAgreementsToRemove.cs

```diff
@@ -18,47 +21,65 @@ namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetAccountEmployerAgreement
 {
     public class WhenIGetAccountEmployerAgreementsToRemove : QueryBaseTest<GetAccountEmployerAgreementsRemoveQueryHandler,GetAccountEmployerAgreementsRemoveRequest, GetAccountEmployerAgreementsRemoveResponse>
     {
+        private Mock<IDasRecruitService> _dasRecruitService;
         private Mock<IEmployerAgreementRepository> _repository;
         private Mock<IHashingService> _hashingService;
         private Mock<IEmployerCommitmentApi> _commitmentsApi;
         public override GetAccountEmployerAgreementsRemoveRequest Query { get; set; }
         public override GetAccountEmployerAgreementsRemoveQueryHandler RequestHandler { get; set; }
         public override Mock<IValidator<GetAccountEmployerAgreementsRemoveRequest>> RequestValidator { get; set; }
-
+        public Vacancy Vacancy { get; set; }
+        IEnumerable<Vacancy> Vacancies { get; set; }
         private const string ExpectedUserId = "456TGFD";
         private const string ExpectedHashedAccountId = "456TGFD";
+        private const string ExpectedHashedAgreementId = "456TGED";
         private const string ExpectedLegalEntityCode = "98TYG123";
+        private const long ExpectedLegalId = 3846;
         private const long ExpectedAccountId = 98172938;
         private const long ExpectedAgreementId = 8765;
+        private const long ExpectedAccountLegalEntityId = 2017;
```

Comment:
Is this used for anything?

Reply:
yes, i am using it in this setup : _repository.Setup(r => r.GetEmployerAgreement(ExpectedAgreementId))

Reply:
It's returned but then never used for anytthing?  There are no set ups or assertions that need to value so it doesn't need to be a constant/

Reply:
This has now been fixed.